openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx

```diff
@@ -299,7 +326,13 @@ const Users = ({
   const getDescriptionComponent = () => {
     if (isAdminUser || isLoggedinUser || isAuthDisabled) {
       return (
-        <div className="tw--ml-5 tw-flex tw-items-center tw-justify-between tw-px-3">
+        <div
+          className={classNames(
+            'tw--ml-5 tw-flex tw-items-center tw-justify-between tw-px-3',
+            {
+              'm-b-md': !isAuthProviderBasic || !isLoggedinUser,
+            }
+          )}>
```

Comment:
Other components should not get affected. You can add spacing to Change password component itself